passkey-api/example-implementation.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Example Implementation
+description: Without using any SDKs, just raw HTTP, how would you use the Passkey API?
+---
+
+This is an example implementation showing how to use the Passkey API with Node.js. (You can use any language you want to, of course!)
+
+However, if you're already using JavaScript/TypeScript for your backend, you can use [\@teamhanko/passkeys-sdk](https://www.npmjs.com/package/@teamhanko/passkeys-sdk), which do all of the below for you.
```

Comment:
The link here still does not redirect to the package

Reply:
Sorry, had to delay this a bit, package will be up very soon

Package name is correct so this will not 404 once the package is up on NPM